static/canjs.js

```diff
@@ -116,13 +117,15 @@ function navigate(href) {
 		return;
 	}
 
+	$articleContainer.scrollTop(0);
```

Comment:
Why is this above the `if(navigating)` condition?  Do we want the user to be able to click a link and have it scroll to the top during the fetching of a new page?